shell/platform/windows/angle_surface_manager.cc

```diff
@@ -205,7 +208,27 @@ bool AngleSurfaceManager::CreateSurface(WindowsRenderTarget* render_target,
 
   EGLSurface surface = EGL_NO_SURFACE;
 
-  const EGLint surfaceAttributes[] = {EGL_NONE};
+  std::vector<EGLint> surface_attributes;
+
+  if (!CompositionSupported()) {
+    // eglPostSubBufferNV during resizing causes significant artifacts on
+    // Windows 7, so use fixed size surface instead
```

Comment:
Since this code is touching the resize logic, I wonder if we should also revert `eglPostSubBufferNV` in the non-UWP composition case as well pending https://github.com/flutter/flutter/issues/76465 being addressed.  There are reports of the visual artifacts in cases where WM_NCCALCSIZE is not in use (see the thread in the bug) which means the mainstream case can be subject to the artifacts.  If you don't want to address in this PR I will do in a follow-on PR.  If the new DComp path in some way depends on `eglPostSubBufferNV` then we may have to hold off on landing this PR until the above big is correctly fixed.  That is currently backed up behind the UWP work.

Reply:
The problem with composition enabled (both direct composition and uwp I believe) and recreating surface during resizing is that every now and then you get brief transparent flashes (even with `ClearContext` in place). I wasn't able to find any workaround for this. So if you decide to revert `eglPostSubBufferNV` then it would be better to NOT apply this PR until there's proper solution for manual surface resizing.

Reply:
I have a possible fix for the resize present issue which I pinged you about.